thunder/torch/__init__.py

```diff
@@ -3489,6 +3489,21 @@ def embedding_backward(grad, indices, num_weights, padding_idx, scale_grad_by_fr
     return result
 
 
+@torchsymbol(torch.nn.functional.one_hot, id="torch.nn.functional.one_hot", is_method=False)
+def one_hot(a: TensorProxy, /, num_classes: int) -> TensorProxy:  # Should we use TensorProxy for a?
```

Comment:
To be consistent with other type annotations in this file the annotation would be:

```
a: TensorLike, /, num_classes: int) -> TensorLike:
```